packages/code-du-travail-frontend/src/modules/modeles-de-courriers/queries.ts

```diff
@@ -27,7 +29,44 @@ export const fetchAllModels = async <
     _source: fields,
     index: elasticDocumentsIndex,
   });
-  return response.hits.hits
-    .map(({ _source }) => _source)
-    .filter((source) => source !== undefined);
+  return response.hits.hits.map(({ _source }) => _source).filter(nonNullable);
```

Comment:
```suggestion
  return response.hits.hits.map(({ _source }) => _source).filter((model) => model !== undefined);
```

La nouvelle version de typescript le supporte maintenant :)

Reply:
t'es sur ?

Reply:
je viens de tester sur ta branche et je n'ai pas d'erreur. Tu as bien mis à jour typescript et relancer ?